package.json

```diff
@@ -100,5 +105,8 @@
       "\\.(jpg|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$": "<rootDir>/__mocks__/fileMock.js",
       "\\.(css|scss)$": "identity-obj-proxy"
     }
+  },
+  "dependencies": {
+    "glob": "^7.1.3"
```

Comment:
@davidjoy: FYI: Something in the precommit hook complained if this is not a real dependency (and only a dev dependency) because it is used in i18n-concat.js. I think this is fine for now, but something to keep in mind for future iterations.

Reply:
There's an annoying problem here in that i18n-concat.js used by the build system/node, but it's being evaluated as being part of the app by the precommit hook.  If we had a way to ignore it, that'd probably be best, as i18n-concat.js itself is a dev/build thing.

Reply:
Right. This is the question I tried to ask the other day without having a specific example. I might be able to try to disable the linter on that line.